profiles/student/ctmartin.md

```diff
@@ -0,0 +1,8 @@
+name: Christian Martin
+blog: https://blog.ctmartin.me/
+email: ctm2142@rit.edu
+major: New Media Interactive Development 2020
```

Comment:
You probably didn't see this since the changes were in #38, but could you please split the graduation field out to a new line?

```
major: New Media Interactive Development
graduation: 2020
```